README.md

```diff
@@ -8,18 +8,41 @@
   <h1>SmartBear MCP server</h1>
 </div>
 
-An [MCP](https://modelcontextprotocol.io) server for SmartBear's API Hub, Test Hub and Insight Hub.
+[![Documentation](https://img.shields.io/badge/documentation-latest-blue.svg)](https://developer.smartbear.com/smartbear-mcp)
 
-## Usage
+A Model Context Protocol (MCP) server that provides AI assistants with seamless access to SmartBear's suite of testing and monitoring tools, including Insight Hub, Reflect, and API Hub.
+
+## What is MCP?
+
+The [Model Context Protocol (MCP)](https://modelcontextprotocol.io/introduction) is an open standard that enables AI assistants to securely connect to external data sources and tools. This server exposes SmartBear's APIs through natural language interfaces, allowing you to query your testing data, analyze performance metrics, and manage test automation directly from your AI workflow.
+
+## Supported Tools
+
+See individual guides for suggested prompts and supported tools and resources:
+
+- [Insight Hub](https://developer.smartbear.com/smartbear-mcp/docs/insight-hub-integration) - Comprehensive error monitoring and debugging capabilities.
+- [Test Hub](https://developer.smartbear.com/smartbear-mcp/docs/test-hub-integration) - Test management and execution capabilities
+- [API Hub](https://developer.smartbear.com/smartbear-mcp/docs/api-hub-integration) - Portal management capabilities
```

Comment:
Should we remove the README.md files inside each product (or replace their content with the above link)?